CONTRIBUTING.md

```diff
@@ -2,16 +2,14 @@
 
 1. Familiarize yourself with the codebase by reading the [docs](docs), in
    particular the [development](contributing/developing.md) doc.
-1. Create a new issue before starting your project so that we can keep track of
-   what you are trying to add/fix. That way, we can also offer suggestions or
-   let you know if there is already an effort in progress.
+1. Create a new discussion before starting your project so that we can keep track of what you are trying to add/fix. If it fits in our roadmap, then we will review your contribution and work with you to get it merged. That way, we can also offer suggestions or let you know if there is already an effort in progress.
```

Comment:
is the discussion to keep track or to validate/analyze if we will spend time on it?

Reply:
Analyze if we will spend time on it - I'll fix that.